src/tests/unit/tests/DetailsView/handlers/nav-link-handler.test.ts

```diff
@@ -74,4 +77,43 @@ describe('NavLinkHandler', () => {
             detailsViewActionMessageCreatorMock.verifyAll();
         });
     });
+
+    describe('onRequirementClick', () => {
+        it('should call selectRequirement and changeRightContentPanel with appropriate params', () => {
+            const requirementLink = {
+                key: 'some requirement',
+                testType: -1,
+            } as TestRequirementLeftNavLink;
+            detailsViewActionMessageCreatorMock
+                .setup(amc =>
+                    amc.selectRequirement(eventStub, requirementLink.key, requirementLink.testType),
+                )
+                .verifiable();
```

Comment:
I usually see `.Times(<expected number of times>)` here. Why is it not needed in this case?

Reply:
I feel like that's not been a hard rule per-say but I can definitely add it here.

Reply:
Actually, this doesn't need it due to the mock being Strict Behavior. For that, it's assumed when you set it up that it will only be called once and to call it more you must explicitly set it up to do so.